.github/workflows/build.yml

```diff
@@ -7,11 +7,41 @@ on:
       - opened
 
 jobs:
-  build:
-    runs-on: macos-latest
+  build-ios:
+    name: Build iOS
+    runs-on: macOS-14
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
+
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: '15.2'
+
     - name: Build iOS
-      run: swift build -Xswiftc "-sdk" -Xswiftc "`xcrun --sdk iphonesimulator --show-sdk-path`" -Xswiftc "-target" -Xswiftc "x86_64-apple-ios13.0-simulator"
-    - name: Build tvOS
-      run: swift build -Xswiftc "-sdk" -Xswiftc "`xcrun --sdk appletvsimulator --show-sdk-path`" -Xswiftc "-target" -Xswiftc "x86_64-apple-tvos13.0-simulator"
+      run: xcodebuild -scheme SimultaneouslyScrollView -destination "platform=iOS Simulator,name=iPhone 15 Pro,OS=latest"
```

Comment:
For simplicity I switched to `xcodebuild` over `swift build` which has very limited documentation.